manifests/0000_31_cluster-baremetal-operator_05_rbac.yaml

```diff
@@ -236,21 +236,19 @@ rules:
   - use
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
 metadata:
+  name: system:serviceaccount:openshift-machine-api:cluster-baremetal-operator
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
-  name: cluster-baremetal-operator
-  namespace: openshift-machine-api
 roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: cluster-baremetal-operator
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
I needed to set this rolebinding to allow access to the configmap in another namespace,
is there a more appropriate way to allow access?
`error: configmaps "cluster-config-v1" is forbidden: User "system:serviceaccount:openshift-machine-api:cluster-baremetal-operator" cannot get resource "configmaps" in API group "" in the namespace "kube-system"`

Reply:
You'll need to add a new rbac rule in this section https://github.com/openshift/cluster-baremetal-operator/blob/master/controllers/provisioning_controller.go#L69.

Reply:
I've added configmaps to the cluster-baremetal-operator ClusterRole which I think is what I should have been doing in the first place.